librz/main/rz-find.c

```diff
@@ -294,11 +295,24 @@ static int rzfind_open_file(RzfindOptions *ro, const char *file, const ut8 *data
 		goto err;
 	}
 
+	RzCore core = { 0 };
+	rz_core_init(&core);
+	RzBinOptions opt;
+	RzBin *bin = core.bin;
+	rz_bin_options_init(&opt, 0, 0, 0, false, 2);
+	RzBinFile *bf = rz_bin_open(bin, file, &opt);
+	if (ro->json) {
+		bf->strmode = RZ_MODE_JSON;
+	} else {
+		bf->strmode = RZ_MODE_SIMPLE;
+	}
```

Comment:
```suggestion
	bf->strmode = ro->json ? RZ_MODE_JSON : RZ_MODE_SIMPLE;
```

I think in this case a ternary operator fits very well and it's very readable.